aoc_day_2/src/part2.rs

```diff
@@ -42,7 +42,6 @@ where P: AsRef<Path>, {
 
 fn calculatewinning_score(outcomes: &mut Vec<&str>) -> u32 {
     
-    
     let mut score: u32 = 0;
     let mut subject_move = outcomes[1];
```

Comment:
_outcomes_ in part two is defined as **[(opponent move), (win/lose or draw)]**
so here you are really not getting a _subject_move_. [Shadowing variables](https://en.wikipedia.org/wiki/Variable_shadowing) is not nearly as frowned upon in rust, but usually the domain of the thing being written to is the same. Here, they are categorically different (you next assign an actual move to _subject_move_).

Consider renaming it here (only) to something else like _subject_goal_.